crowbar_framework/app/controllers/nodes_controller.rb

```diff
@@ -239,8 +239,22 @@ def group_change
 
       node.save
 
-      Rails.logger.info "Node #{node.name} (#{node.alias}) changed its group to be #{node.group || "automatic"}."
-      render inline: "Added #{node.name} to #{node.group}.", cache: false
+      respond_to do |format|
+        format.html do
+          flash[:success] = I18n.t(
+            "nodes.group_change.updated",
+            name: node.name,
+            group: node.group
```

Comment:
Hrm, sorry, I should have seen that earlier, but: should we do something like `node.group || "automatic"` which was in the previous logging? It seems it might be needed.